### ISSUE TITLE
Conda not installing r-essentials

### ISSUE THREAD

**Issue by seleregb** [conda/conda#3023]
i get this error after executing command in console 

> conda install -c r r-essentials

```
Traceback (most recent call last):
  File "C:\Users\Tosin\Anaconda2\Scripts\conda-script.py", line 5, in <module>
    sys.exit(main())
  File "C:\Users\Tosin\Anaconda2\lib\site-packages\conda\cli\main.py", line 120, in main
    exit_code = args_func(args, p)
  File "C:\Users\Tosin\Anaconda2\lib\site-packages\conda\cli\main.py", line 130, in args_func
    exit_code = args.func(args, p)
  File "C:\Users\Tosin\Anaconda2\lib\site-packages\conda\cli\main_install.py", line 69, in execute
    install(args, parser, 'install')
  File "C:\Users\Tosin\Anaconda2\lib\site-packages\conda\cli\install.py", line 407, in install
    execute_actions(actions, index, verbose=not args.quiet)
  File "C:\Users\Tosin\Anaconda2\lib\site-packages\conda\plan.py", line 595, in execute_actions
    inst.execute_instructions(plan, index, verbose)
  File "C:\Users\Tosin\Anaconda2\lib\site-packages\conda\instructions.py", line 136, in execute_instructions
    cmd(state, arg)
  File "C:\Users\Tosin\Anaconda2\lib\site-packages\conda\instructions.py", line 59, in EXTRACT_CMD
    extract(arg)
  File "C:\Users\Tosin\Anaconda2\lib\site-packages\conda\install.py", line 852, in extract
    exp_backoff_fn(os.rename, temp_path, path)
  File "C:\Users\Tosin\Anaconda2\lib\site-packages\conda\install.py", line 236, in exp_backoff_fn
    result = fn(*args)
WindowsError: [Error 5] Access is denied
```

**Comment by kalefranz** [comment]
Can you please try again?  I'm guessing you're on a not-exactly-speedy Windows machine with a virus scanner running. We're aware of this issue, and we need to increase some timeouts so that we don't have to tell you to disable your virus scanner.

**Comment by seleregb** [comment]
okay. i'll try again. 

this is the result of the 

> conda info

```
platform : win-64
        conda version : 4.1.4
    conda-env version : 2.5.1
  conda-build version : 1.21.2
       python version : 2.7.11.final.0
     requests version : 2.10.0
     root environment : C:\Users\Tosin\Anaconda2  (writable)
  default environment : C:\Users\Tosin\Anaconda2
     envs directories : C:\Users\Tosin\Anaconda2\envs
        package cache : C:\Users\Tosin\Anaconda2\pkgs
         channel URLs : https://repo.continuum.io/pkgs/free/win-64/
                        https://repo.continuum.io/pkgs/free/noarch/
                        https://repo.continuum.io/pkgs/pro/win-64/
                        https://repo.continuum.io/pkgs/pro/noarch/
          config file : None
         offline mode : False
    is foreign system : False
```

**Comment by mingwandroid** [comment]
I *think* conda 4.1.4 had used a millisecond timescale for the timeout, so you probably want to update conda and then try again.

**Comment by seleregb** [comment]
Okay i'll try that                                                                                                                                                                                                                                                                                                                                         Sent from my BlackBerry 10 smartphone.                                                                                                                                                                                                                From: Ray DonnellySent: Saturday, 9 July 2016 03:03To: conda/condaReply To: conda/condaCc: Gbenro Selere; AuthorSubject: Re: [conda/conda] Conda not installing r-essentials (#3023)I think conda 4.1.4 had used a millisecond timescale for the timeout, so you probably want to update conda and then try again.

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.

**Comment by seleregb** [comment]
I tried to update conda itself but still go an access denied error 

> conda update conda

```
Fetching package metadata .......
Solving package specifications: ..........

Package plan for installation in environment C:\Users\Tosin\Anaconda2:

The following packages will be UPDATED:

    conda: 4.1.4-py27_0 --> 4.1.6-py27_0

Proceed ([y]/n)? y

Extracting packages ...
An unexpected error has occurred, please consider sending the            |   0%
following traceback to the conda GitHub issue tracker at:

    https://github.com/conda/conda/issues

Include the output of the command 'conda info' in your report.


Traceback (most recent call last):
  File "C:\Users\Tosin\Anaconda2\Scripts\conda-script.py", line 5, in <module>
    sys.exit(main())
  File "C:\Users\Tosin\Anaconda2\lib\site-packages\conda\cli\main.py", line 120, in main
    exit_code = args_func(args, p)
  File "C:\Users\Tosin\Anaconda2\lib\site-packages\conda\cli\main.py", line 130, in args_func
    exit_code = args.func(args, p)
  File "C:\Users\Tosin\Anaconda2\lib\site-packages\conda\cli\main_update.py", line 64, in execute
    install(args, parser, 'update')
  File "C:\Users\Tosin\Anaconda2\lib\site-packages\conda\cli\install.py", line 407, in install
    execute_actions(actions, index, verbose=not args.quiet)
  File "C:\Users\Tosin\Anaconda2\lib\site-packages\conda\plan.py", line 595, in execute_actions
    inst.execute_instructions(plan, index, verbose)
  File "C:\Users\Tosin\Anaconda2\lib\site-packages\conda\instructions.py", line 136, in execute_instructions
    cmd(state, arg)
  File "C:\Users\Tosin\Anaconda2\lib\site-packages\conda\instructions.py", line 59, in EXTRACT_CMD
    extract(arg)
  File "C:\Users\Tosin\Anaconda2\lib\site-packages\conda\install.py", line 852, in extract
    exp_backoff_fn(os.rename, temp_path, path)
  File "C:\Users\Tosin\Anaconda2\lib\site-packages\conda\install.py", line 236, in exp_backoff_fn
    result = fn(*args)
WindowsError: [Error 5] Access is denied
```

Could it be because I don't have a really fast internet connection??

**Comment by kalefranz** [comment]
Mult-user support was fixed with conda 4.3.
